docs/beta-launch/avasummit/overview.md

```diff
@@ -0,0 +1,127 @@
+```
+
+<div class="remix-callout">
+      <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/beta-launch/avasummit/VRFv2SubscriptionManager.sol" target="_blank" >Open in Remix</a>
+      <a href="/docs/conceptual-overview/#what-is-remix">What is Remix?</a>
+</div>
+
+These parameters define how your requests will be processed:
+
+- `uint64 s_subscriptionId`: The subscription ID that this contract uses for funding requests.
+
+- `address vrfCoordinator`: The address of the Chainlink VRF Coordinator contract. `0x2eD832Ba664535e5886b75D64C46EB9a228C2610` is the Avalanche Fuji Coordinator address.
+
+- `address link`: The LINK token address for your selected network. `0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846` is the Avalanche Fuji Link token address.
+
+- `bytes32 keyHash`: The gas lane key hash value, which is the maximum gas price you are willing to pay for a request in wei. It functions as an ID of the off-chain VRF job that runs in response to requests. Only `0x354d2f95da55398f44b7cff77da56283d9c6c829a4bdf1bbcaf2ad6a4d081f61` is currently available on Avalanche Fuji Testnet.
+
+- `uint32 callbackGasLimit`: The limit for how much gas to use for the callback request to your contract's `fulfillRandomWords()` function. It must be less than the `maxGasLimit` limit on the coordinator contract, `2500000` on Avalanche Fuji Testnet. In this example, the `fulfillRandomWords()` function stores two random values, which cost about 20,000 gas each, so a limit of `100000` gas is sufficient. Adjust this value for larger requests depending on how your `fulfillRandomWords()` function processes and stores the received random values. If your `callbackGasLimit` is not sufficient, the callback will fail, and the contract subscription is still charged for the work done to generate your requested random values.
+
+- `uint16 requestConfirmations`: How many confirmations the Chainlink node should wait before responding. The longer the node waits, the more secure the random value is. It must be greater than the `minimumRequestBlockConfirmations` limit on the coordinator contract, which is `1` on Avalanche Fuji Testnet. In this example, we set the value to `1,` but you can set it higher.
+
+- `uint16 numWords`: How many random values to request. If you can use several random values in a single callback, you can reduce the amount of gas you spend per random value. The total cost of the callback request depends on how your `fulfillRandomWords()` function processes and stores the received random values, so adjust your `callbackGasLimit` accordingly. Here we are requesting two random values.
+
+To use this contract:
+
+1. Open the [contract in Remix](https://remix.ethereum.org/#url=https://docs.chain.link/samples/beta-launch/avasummit/VRFv2SubscriptionManager.sol).
+
+1. Compile and deploy the contract using the Injected Web3 environment. The contract includes all the configuration variables you need, but you can edit them if necessary (cf. above).
```

Comment:
```suggestion
1. Compile and deploy the contract using the Injected Web3 environment. The contract includes all the configuration variables you need, but you can edit them if necessary.
```
They'll have the example open in Remix, so they probably don't need the cf. I recommend avoiding cf. and e.g. when possible.